twitter_clone.css

```diff
@@ -0,0 +1,206 @@
+body {
```

Comment:
Good job overall! Here are few feedback of things that I noticed: 
1. Margin top for login input boxes. Original site has little margin above input boxes for login. 
2. Also maybe giving border-radius for input boxes would be nice. 
3. Your website responds well to user re-sizing the browser size. Well done. 
4. Nice touch with favicon stuff. 
Other than those minor feedbacks, good job with the assignment. Keep coding!